features/networking/multus.feature

```diff
@@ -1263,3 +1263,36 @@ Feature: Multus-CNI related scenarios
       | /usr/sbin/ip | route         |
     Then the output should contain:
       | 192.168.10.0                 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-30054
+  @admin
+  Scenario: Multus namespaceIsolation should allow references to CRD in the default namespace
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    # Create the net-attach-def via cluster admin
+    Given I switch to cluster admin pseudo user
```

Comment:
I suggest you to remove 1273 and 1274 and add after 1278 following
```| ["metadata"]["namespace"] | default |```

This will shorten the scenario. We switch to admin pseudo if we really have series of multiple commands to be executed administratively

Reply:
@anuragthehatter Good idea, fixed it, thanks!